nixos/modules/services/security/vaultwarden/default.nix

```diff
@@ -57,6 +57,14 @@ in {
       '';
     };
 
+    stateDir = mkOption {
+      type = str;
+      default = "/var/lib/bitwarden_rs";
+      description = ''
+        The directury in which vaultwarden will keep its state.
+      '';
```

Comment:
Maybe this:
```suggestion
      description = ''
        The directury in which vaultwarden will keep its state. If left as the default value
          this directory will automatically be created before the vaultwarden server starts, otherwise
          the sysadmin is responsible for ensuring the directory exists with appropriate ownership
          and permissions.
      '';
```